framework/wazuh/tests/test_manager.py

```diff
@@ -53,7 +72,11 @@ def test_restart_ok(test_manager):
     ('input_lists_file', 'output_lists_file')
 ])
 @patch('wazuh.common.ossec_path', test_data_path)
-def test_upload_file(test_manager, input_file, output_file):
+@patch('time.time', lambda: 0)
```

Comment:
I think the lambda function is not really necessary. You could replace that by `return_value=0`.

Reply:
Done here: https://github.com/wazuh/wazuh/pull/2718/commits/b4d6d3b6e8919708414a02f09eff8d463b10850c